learn/polkadot/parachains.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Parachains and parathreads
+description: Introduction to the basic concepts of a parachain and parathreads
+---
+
+# Parachains and Parathreads in Polkadot
+
+## Parachains
+
+Polkadot is a layer zero blockchain that provides essential services to the other chains within the ecosystem: the Parachains.
+
+More specialization leads to better performance, and parachains are sovereign blockchains with high specialization that are interconnected to each other through the relay chain. 
+This model allows each parachain to process transactions in parallel with its own set of collators producing blocks, the performance and the throughput of the network.
+
+In addition to performance gains to the whole network, parachains can communicate with one another, enabling the transfer of assets and/or data and facilitating interoperability while relying on the security model offered by the relay chain.
+
+In short, a parachain is an independent blockchain operating within the Polkadot network, leveraging the interoperability, scalability, shared security and governance features provided by the Polkadot ecosystem. This empowers developers to create specialized blockchains that seamlessly interact with other parachains.
+
+### How Can I Get My Parachain Up and Running ?
+
+To add a new parachain to the Polkadot ecoystem, it is necessary to obtain a slot through the slot auction process. 
+Here's a general overview of what to do:
+
+1. Prepare your parachain: Develop and prepare your parachain project, including its functionality, features, and any necessary code implementation.
```

Comment:
We don't typically use `:` for list items and don't add punctuation at the end of them either